Makefile

```diff
@@ -4,13 +4,13 @@ SHELL := /usr/bin/env bash
 REGISTRY ?= quay.io
 REPOSITORY ?= $(REGISTRY)/openshift/must-gather-operator
 
+# Boilerplate Makefile includes
+include boilerplate/generated-includes.mk
+
 # Include shared Makefiles
-include project.mk
```

Comment:
Should this line be removed here considering https://issues.redhat.com/browse/OSD-4193 has not been completed? This seems like it will interfere with our ability to deploy via app-interface's CICD pipelines